CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.17.0
+
+NEW FEATURES:
+
+* The `consul_config_entry` now supports specifying the name of the peered cluster to export the service to.
+
```

Comment:
I don't see a code change in the PR related to that, is that done in a prior PR?

Reply:
The handling of config entries in the provider is a bit special: we call `consulapi.DecodeConfigEntry()` at https://github.com/hashicorp/terraform-provider-consul/blob/master/consul/resource_consul_config_entry.go#L150. This way we don't need to make changes to the provider when there is changes to them, we just update the module used by the provider and the new version of `consulapi.DecodeConfigEntry()` is called.